### DISCUSSION TITLE
Making a version for Windows 11

### DISCUSSION THREAD

**Discussion by dprooney1982** [vincentneo/LosslessSwitcher#70] [Ideas]
Since Apple released a stand-alone version of Apple Music for Windows, it would be great to have this useful tool for the Windows OS as well.

**Comment by vincentneo** [comment]
I haven't really tried Windows 11 at all, actually. (didn't bother with the additional troubles regarding compatibility)

The last time I tried playing back Apple Music content from Windows was via iTunes. (which if my memory serves me correct, has no lossless playback functionality). How much/little has things changed there?

**Reply by Robertsmania** [reply]
I don’t have a Windows 11 machine at the moment either, so I’m a little reluctant to reply without personal experience, but the current preview app does support lossless and high-res.

https://www.audiosciencereview.com/forum/index.php?threads/apple-music-lossless-is-finally-coming-to-windows-11.40922/

As far as rate switching, it sounds like it’s similar to the current MacOS where iTunes is not WASAPI exclusive and will not adjust the rate/bit-depth settings to match the content.  The Apple docs say you can select the values you want, whether it uses Direct Sound or WAS but recommends using the same values you have in the system audio settings (but it’s also likely those support docs are not talking about features in the preview versions).

![25B6F066-EA05-4B9A-ABD0-6F02A1BC809B](https://user-images.githubusercontent.com/11642124/218318732-01721b25-cb5c-44bb-9ea0-3d4e1e164b95.png)

**Reply by dprooney1982** [reply]
To me it looks like the preview Windows 11 app works just like the version for MacOS as mentioned above. I am not sure but I was hoping the implementation would be the same.
